packages/controller/src/resources/tenants/index.ts

```diff
@@ -15,10 +15,53 @@
  */
 
 import { KubeConfig } from "@kubernetes/client-node";
-import { ResourceCollection, Namespace } from "@opstrace/kubernetes";
+import { ResourceCollection, Namespace, Secret } from "@opstrace/kubernetes";
 import { State } from "../../reducer";
 import { getTenantNamespace } from "../../helpers";
 
+const CERT_SECRET_NAMESPACE = "ingress"
+const CERT_SECRET_NAME = "https-cert"
```

Comment:
Should I rather pass these infos via a parameter, like [here](https://github.com/opstrace/opstrace/blob/main/packages/controller/src/tasks/reconciliation.ts#L82)?

Reply:
Yes, please pass via parameters.